databuilder/databuilder/extractor/es_base_extractor.py

```diff
@@ -55,6 +60,20 @@ def _get_indexes(self) -> Dict:
 
         return result
 
+    def _get_index_creation_date(self, index_metadata: Dict) -> Optional[float]:
+        if index_metadata is None:
+            return None
+        settings = index_metadata.get('settings', dict())
+        if len(settings) == 0:
+            return None
+        index = settings.get('index', dict())
+        if len(index) == 0:
+            return None
+        creation_date = index.get('creation_date')
```

Comment:
so if creation_date is not none we shall return it. in any other case we return None. could this section be written in a less complex approach?

Reply:
Yes, I'll rewrite it. I wrote it this way to get around some mypy errors, but I agree this could be written more simply.